packages/loader/driver-utils/src/blobAggregationStorage.ts

```diff
@@ -312,7 +312,7 @@ export class BlobAggregationStorage extends SnapshotExtractor implements IDocume
             return summary;
         }
         // Only pack at data store level.
-        const startingLevel = level === 1;
+        const startingLevel = level === 2;
```

Comment:
Consider declaring a const called `dataStoreLevel` or similar and use that here and during unpacking so they are in sync.